app/client/templates/views/send.js

```diff
@@ -208,24 +227,6 @@ Template['views_send'].onRendered(function(){
 
 Template['views_send'].helpers({
     /**
-    React on the template data context
-
-    @method (reactiveData)
-    */
-    'reactiveData': function(deployContract){
```

Comment:
Im not sure if removing this doesn't break anything, as it makes sure that those values are set when reactive changes are incoming anytime the template is open.

I think there was a case where it switches from this template in contractDeploy mode to normal send page, and where this was necessary to be switched and reacting on template state changes..